tests/utils.go

```diff
@@ -227,17 +227,6 @@ func getRunningPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, names
 	return libpod.GetRunningPodByLabel(string(vmi.GetUID()), v1.CreatedByLabel, namespace, vmi.Status.NodeName)
 }
 
-func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance) *k8sv1.Pod {
-	pods, err := getPodsByLabel(string(vmi.GetUID()), v1.CreatedByLabel, vmi.Namespace)
-	util2.PanicOnError(err)
-
-	if len(pods.Items) != 1 {
-		util2.PanicOnError(fmt.Errorf("found wrong number of pods for VMI '%v', count: %d", vmi, len(pods.Items)))
-	}
-
-	return &pods.Items[0]
-}
-
 func getPodsByLabel(label, labelType, namespace string) (*k8sv1.PodList, error) {
```

Comment:
Is it used now?

Reply:
no it's unused now
should I remove this as well?

Reply:
Yes